client/app/src/time/TimeService.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+  'use strict';
+
+  angular.module('arkclient')
+    .service('timeService', ['$http', TimeService]);
+
+  /**
+   * TimeService
+   * @constructor
+   */
+  function TimeService($http) {
+    this.getTime = function() {
+      $http.get('http://date.jsontest.com/', { timeout: 2000 })
+        .then(function(resp) {
+          var date = new Date(Date.parse(resp.data["date"]+" "+resp.data["time"]));
+          //GMT-4 New York Time
+          date.setHours(date.getHours() - 4);
+          return date
```

Comment:
mmm the issue here is that it return a promise and not the real date, also date should be UTC

Reply:
What do u mean with promise?
Every Server which is Not in your Hand is a promise. Gmt is actually the same as utc but the URL can also be changed to a utc one